assets/service_broker/setup_new_broker.rb

```diff
@@ -104,9 +91,17 @@ def enable_service_access
 end
 
 uniquify_config
-push_broker(broker_name, domain)
 
-url = "http://#{broker_name}.#{domain}"
+push_broker(broker_name)
+
+app_guid, routes_object, url = ""
+IO.popen("cf app #{broker_name} --guid") do |cmd|
+  cmd.each { |line| app_guid = "#{line.chop}" }
```

Comment:
`cmd.each` reads a bit strangely here since the expected output is a single app guid. Also, I believe that `chomp` is generally recommended over `chop` when the goal is to remove any record separators, which I assume is the case here.

Reply:
Would you be opposed to something more like this:

```ruby
app_guid = cmd.read.chomp
```